addons/binding/org.openhab.binding.wso2iots/README.md

```diff
@@ -0,0 +1,87 @@
+# wso2iots Binding
+
+This binding is used to connect with a locally hosted WSO2 IoT Server 3.1.0 (https://wso2.com/iot) and get temperature, humidity, light, motion data from the devices enrolled in the WSO2 IoTS building monitor plugin. The binding allows communication between the openHAB server and WSO2 IoT server.
+
+WSO2 is an opensource middleware company and IoTS is one of the five products offered by the company.(https://wso2.com/)
+
+To use the binding, first the user have to locally setup the WSO2 IoTS (https://wso2.com/iot) and deploy the building monitor plugin (https://github.com/wso2/samples-iots/tree/master/floor-analytics-demo). The building monitor agent uses ESP8266 12E, DHT 11, PIR SR501, LM393. (https://github.com/wso2/samples-iots/tree/master/floor-analytics-demo/BuildingMonitorDevice). 
```

Comment:
have -> has

Reply:
It would look more smooth, if you hide urls in links (e.g. [the building monitor plugin](https://github.com/wso2/samples-iots/tree/master/floor-analytics-demo))

Reply:
uses ... LM393 devices